prawns/staking/package.json

```diff
@@ -5,10 +5,9 @@
   "private": true,
   "scripts": {
     "db:init": "docker-compose -f ../../docker-compose.dev.yml down && docker-compose -f ../../docker-compose.dev.yml up -d && yarn build && yarn migrate",
-    "dev:polkadot": "node -r dotenv/config lib/processors/polkadotProcessor.js",
-    "dev:kusama": "node -r dotenv/config lib/processors/kusamaProcessor.js",
-    "process:polkadot": "node lib/processors/polkadotProcessor.js",
-    "process:kusama": "node lib/processors/kusamaProcessor.js",
+    "dev:polkadot": "NETWORK=polkadot node -r dotenv/config lib/processor.js",
```

Comment:
this is just opinion, but I'd argue we don't need these, as we need a local .env to run anyway to set the DB port

Reply:
But you might want to use the same db to test both? Rather than keep changing the env file when developing?